content/blog/2022-01-14-users-guide-to-contribution-ross/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "A contributor's guide to the Galaxy"
+tease: "Contribution opportunities"
+authors: "Ross Lazarus"
+external_url: ""
+date: "2022-01-14"
+source_blog: ""
+source_blog_url: ""
+autotoc: false
+---
+
+Galaxy is an open science computing project, with a large and active global community, that invites users and developers to join in and help make the project even better.
+
+You can use Galaxy without paying the project. It is open source, so instead of charging a fee, the project offers opportunities to contribute your expertise and skills instead. A large part of Galaxy's success comes from the hundreds of individuals who have chosen to join the [inclusive and safe](https://galaxyproject.org/community/coc/) global collaboration, making transparent, shareable, open science computing readily available to any researcher.
+
+Contributor motivations vary. Some work on Galaxy for a living. Others want to pay it forward or perhaps scratch an open source itch.
+
+Whatever motivates individuals, in addition to the satisfaction of contributing to an important open science resource, they gain valuable professional experience and skills, collaborating with talented and friendly people, working on problems they care about.
+
+**If this sounds interesting, please read on to learn more about contributing to Galaxy**
```

Comment:
```suggestion
**If this sounds interesting, please read on to learn more about contributing to Galaxy.**
```

Reply:
We have now this page: https://galaxyproject.org/community/contributing/ Do you think that we could link it here or somewhere else?